### PR TITLE
theme: Add vscode-monokai-charcoal

### PR DESCRIPTION
**Pull request by d1y**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 [submodule "extensions/nu"]
 	path = extensions/nu
 	url = https://github.com/zed-extensions/nu
+[submodule "extensions/vscode-monokai-charcoal"]
+	path = extensions/vscode-monokai-charcoal
+	url = git@github.com:d1y/vscode-monokaicharcoal.zed.git
```
[the remark this answers is not in the document]
Comment (maxdeviant) [reply .gitmodules:15]: I don't think it matters which ones we use, as we're just `git clone`ing them.

**Comment by maxdeviant** [comment]
@d1y Looks like the extension repository as a typo in the `extension.json` filename: https://github.com/d1y/vscode-monokaicharcoal.zed/blob/7652f27cfc1446199f54ce4abcd758d1f9a41379/extenstion.json

It should be `extension.json`, but it is currently `extenstion.json`.

**Comment by d1y** [comment]
My bad. focus push

**Comment by maxdeviant** [comment]
@d1y Thank you!

**Pull request merged**